moveit_demo_nodes/run_moveit_cpp/CMakeLists.txt

```diff
@@ -1,17 +1,13 @@
 cmake_minimum_required(VERSION 3.5)
 project(run_moveit_cpp)
 
-# Default to C++14
-if(NOT CMAKE_CXX_STANDARD)
-  set(CMAKE_CXX_STANDARD 14)
-endif()
-
 if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
-  add_compile_options(-Wall -Wextra -Wpedantic)
+  add_compile_options(-Wpedantic)
 endif()
 
 # find dependencies
 find_package(ament_cmake REQUIRED)
+find_package(moveit_common REQUIRED)
```

Comment:
Shouldn't we just add this and the `moveit_package()` macro to the top in all CMakeLists files?